viewer/addUser.js

```diff
@@ -132,7 +132,16 @@ if (process.argv.length < 5) {
   help();
 }
 
+// Use Xinsecure instead of insecure as node
+// consumes the --insecure flag anywhere in the command line
```

Comment:
You just need to do ```node addUser.js -c ../tests/config.test.ini -n testuser --insecure admin admin admin --admin --packetSearch```

Reply:
will do!  i missed this in the merge as always